config/default

```diff
@@ -1,11 +1,9 @@
-export instanceid = main
-export base_url = /
+export instanceid ?= main
+export base_url ?= /
+export ui_url ?= "TO BE DEFINED"
```

Comment:
What about using "http://c2corgv6.demo-camptocamp.com" here?
It would be consistent with 
```
export image_backend_url = http://c2corgv6-images.demo-camptocamp.com
export image_url = http://c2corgv6-images.demo-camptocamp.com/active/

export discourse_url = http://c2corgv6-discourse.demo-camptocamp.com
```
a few lines below.